Add try-catch to VM-Pip-Install

sara-rn opened the pull request:
Remove try-catch from install scripts that use VM-Pip-Install
Closes #1205

Ana06 reviewed (changes requested):
Thanks for working on this @sara-rn.

[nitpick] The commit message is a bit confusing and it does not explain **why** the change is needed. I propose to improve it as follows:

```
Add try-catch to VM-Pip-Install

Add missing `try-catch` to `VM-Pip-Install` so that packages using this
function (such as ida.plugin.hashdb.vm) don't have to add a try-catch. 
```

sara-rn commented:
@Ana06 thanks for the review, I changed the ida.plugin.capa.vm nuspec version back to the previous one because the chocolatelyinstall was also changed to a previous state but then of course the lint complains now cause I haven't changed the version

sara-rn commented:
@Ana06 changes have been addressed, thanks for the review

Ana06 reviewed (approved):
Thanks @sara-rn!
